finitio.gemspec

```diff
@@ -1,186 +1,39 @@
-
-  # The list of author names who wrote this gem.
-  #
-  # If you are providing multiple authors and multiple emails they should be
-  # in the same order.
-  #
   s.authors = ["Bernard Lambeau"]
-
-  # Contact emails for this gem
-  #
-  # If you are providing multiple authors and multiple emails they should be
-  # in the same order.
-  #
-  # NOTE: Somewhat strangly this attribute is always singular!
-  #       Don't replace by s.emails = ...
   s.email  = ["blambeau@gmail.com"]
-
-  ################################################################### PATHS, FILES, BINARIES
-
-  # Paths in the gem to add to $LOAD_PATH when this gem is
-  # activated (required).
-  #
-  # The default 'lib' is typically sufficient.
   s.require_paths = ["lib"]
-
-  # Files included in this gem.
-  #
-  # By default, we take all files included in the Manifest.txt file on root
-  # of the project. Entries of the manifest are interpreted as Dir[...]
-  # patterns so that lazy people may use wilcards like lib/**/*
-  #
   here = File.expand_path(File.dirname(__FILE__))
   s.files = File.readlines(File.join(here, 'Manifest.txt')).
                  inject([]){|files, pattern| files + Dir[File.join(here, pattern.strip)]}.
                  collect{|x| x[(1+here.size)..-1]}
 
-  # Test files included in this gem.
-  #
   s.test_files = Dir["test/**/*"] + Dir["spec/**/*"]
-
-  # The path in the gem for executable scripts (optional)
-  #
   s.bindir = "bin"
-
-  # Executables included in the gem.
-  #
   s.executables = (Dir["bin/*"]).collect{|f| File.basename(f)}
-
-  ################################################################### REQUIREMENTS & INSTALL
-  # Remember the gem version requirements operators and schemes:
-  #   =  Equals version
-  #   != Not equal to version
-  #   >  Greater than version
-  #   <  Less than version
-  #   >= Greater than or equal to
-  #   <= Less than or equal to
-  #   ~> Approximately greater than
-  #
-  # Don't forget to have a look at http://lmgtfy.com/?q=Ruby+Versioning+Policies
-  # for setting your gem version.
-  #
-  # For your requirements to other gems, remember that
-  #   ">= 2.2.0"              (optimistic:  specify minimal version)
-  #   ">= 2.2.0", "< 3.0"     (pessimistic: not greater than the next major)
-  #   "~> 2.2"                (shortcut for ">= 2.2.0", "< 3.0")
-  #   "~> 2.2.0"              (shortcut for ">= 2.2.0", "< 2.3.0")
-  #
-  s.add_dependency("citrus", ">= 2.4", "< 4.0")
-
-  #
-  # One call to add_dependency('gem_name', 'gem version requirement') for each
-  # runtime dependency. These gems will be installed with your gem.
-  # One call to add_development_dependency('gem_name', 'gem version requirement')
-  # for each development dependency. These gems are required for developers
-  #
-  # We use Gemfile for development dependencies.
   
+  s.add_dependency("citrus", ">= 3.0", "< 4.0")
+  s.add_dependency("multi_json", "~> 1.15")
```

Comment:
WHY this one?

Reply:
@blambeau  multi_json is used https://github.com/blambeau/finitio-rb/blob/85983bccd3807ee31be4deaf39688c439d94a723/features/support/env.rb#L5 in the app and seems no longer supported by cucumber.
![Capture d’écran de 2021-01-07 15-18-45](https://user-images.githubusercontent.com/22132947/103909991-3e2c3f80-50fc-11eb-8c07-7616dcf3c782.png)
![Capture d’écran de 2021-01-07 15-18-18](https://user-images.githubusercontent.com/22132947/103909997-3f5d6c80-50fc-11eb-872a-0349e0ad6024.png)

By Updating Cucumber we needed to add multi_json to dependencies.

Reply:
Ok @ciremoussadia, but the `features` folder is for cucumber tests. The dependency can then be moved to the development group, instead of a runtime dev.

Reply:
Sure